.circleci/config.yml

```diff
@@ -30,13 +31,20 @@ commands:
           key: source-v1-{{ .Branch }}-{{ .Revision }}
           paths:
             - ".git"
-  npm_install:
-    description: "Install npm modules"
+  npm_install_from_cache:
+    description: "npm install and save cache"
     steps:
+      - restore_cache:
+          keys:
+            - node-{{ checksum "package-lock.json" }}
+            - node-
       - run:
-          name: Bootstrap
+          name: Install npm packages
           command: npm run bootstrap
-
+      - save_cache:
+          key: node-{{ checksum "package-lock.json" }}
+          paths:
+            - "node_modules"
```

Comment:
added, `/packages/*/node_modules` as npm cache in circle config

Reply:
@shudarshon-binary thanks, but it also needs to only do this if the checksum of the `./packages/*/package-lock.json` is also the same, otherwise that means one of the sub `node_modules` is different, and the build will fail.